index.js

```diff
@@ -66,10 +86,26 @@ class PluginXrpPaychan extends PluginBtp {
       }]
     }
 
+    if (protocolMap.info) {
+      debug('got info request from peer')
+
+      return [{
+        protocolName: 'info',
+        contentType: BtpPacket.MIME_APPLICATION_JSON,
+        data: Buffer.from(JSON.stringify({
+          currencyScale: this._currencyScale
+        }))
+      }]
+    }
+
     if (!this._dataHandler) {
       throw new Error('no request handler registered')
     }
 
+    if (!ilp) {
+      throw new Error('no ilp protocol on request')
```

Comment:
Errors thrown here aren't caught anywhere

Reply:
`ilp-plugin-btp` is catching errors thrown by `_handleData` or `_handleMoney`. It translates the errors into a BTP error packet and sends it to the caller.